public/javascripts/continuousPlay.js

```diff
@@ -1,5 +1,11 @@
 /* eslint-disable no-unused-vars */
 
+const pageNum = parseInt(document.getElementById("pageNum").value);
+console.log(pageNum);
+if (pageNum === 1) {
+  console.log(document.getElementById("button-back"));
```

Comment:
Ignore the console.logs here for now, as i started out here planning to implement the "back" button, but couldn't work on it without changing how the play page is being rendered first.